scripts/devaccess/devaccess_users.yaml

```diff
@@ -0,0 +1,14 @@
+roles:
+- name: ALL
+  commands:
+  - oc get nodes -ndefault
+  - oc get nodes -ndefault -ojson
+  - rpm -qa
+- name: super_role
+  commands:
+  - uname
+
+users:
+- username: superperson
```

Comment:
This is granting access to a randomly named user.  Possible to include this block commented out, or inactive to demonstrate format without granting privs to a random user?